contracts/src/WarpProtocolRegistry.sol

```diff
@@ -37,8 +36,10 @@ abstract contract WarpProtocolRegistry {
     uint256 internal _latestVersion;
 
     // Mappings that keep track of the protocol version and corresponding contract address.
-    mapping(uint256 => address) internal _versionToAddress;
-    mapping(address => uint256) internal _addressToVersion;
+    mapping(uint256 version => address protocolAddress)
```

Comment:
since `WarpProtcoolRegistry` is also abstract could consider changing visiblity for the state variables too, but we are looking to remove this contract in favor of `TeleporterRegistry` anyways